WordPress/src/main/java/org/wordpress/android/ui/accounts/NewUserFragment.java

```diff
@@ -488,8 +490,14 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         termsOfServiceTextView.setOnClickListener(new OnClickListener() {
                                                       @Override
                                                       public void onClick(View v) {
+                                                          Resources res = getResources();
+                                                          Configuration conf = res.getConfiguration();
+                                                          String localString = conf.locale.toString();
+                                                          if (localString.contains("_")) {
+                                                              localString = localString.substring(0, localString.indexOf("_"));
+                                                          }
```

Comment:
We should use `LanguageUtils.getPatchedCurrentDeviceLanguage(v.getContext())` here to get the correct language string.